### PR TITLE
PBS Bid Adapter: outstream video renderer bugfix

### PR DESCRIPTION
**Pull request by ChrisHuie**
From Issue #7037 . Checks that at least 1 renderer is speciified

### CODE REVIEW

File: modules/prebidServerBidAdapter/index.js [hunk 1]
```diff
@@ -607,7 +607,7 @@ const OPEN_RTB_PROTOCOL = {
       }
 
       if (!utils.isEmpty(videoParams)) {
-        if (videoParams.context === 'outstream' && (!videoParams.renderer || !adUnit.renderer)) {
+        if (videoParams.context === 'outstream' && (!videoParams.renderer && !adUnit.renderer)) {
```
Comment (muuki88) [inline modules/prebidServerBidAdapter/index.js:610]: Maybe that double negating was causing the confusion :smile: 

What do you think of:

```suggestion
        if (videoParams.context === 'outstream' && (videoParams.renderer || adUnit.renderer)) {
```

**Review by muuki88 — COMMENTED** [review]
Maybe a unit test to cover this?

**Comment by ChrisHuie** [comment]
> Maybe a unit test to cover this?

Good. Idea. I will add one today  :)

**Review by karimMourra — APPROVED** [review]

**Review by patmmccann — APPROVED** [review]

**Comment by karimMourra** [comment]
LGTM

**Comment by patmmccann** [comment]
Covered in video taskforce today, thanks for the really important bugfix @ChrisHuie !

**Comment by patmmccann** [comment]
Take a look at #7208 for tests

**Pull request merged**